apps/console/src/extensions/configs/models/scim.ts

```diff
@@ -18,6 +18,10 @@
 
 export interface SCIMConfigInterface {
 
```

Comment:
Lets remove the line breaks. Check in the other interface as well.

Reply:
Fixed with 03620d3af2411722ce3bc45926ed437690cbef1d